src/logonserver/Realm/RealmManager.hpp

```diff
@@ -30,8 +30,8 @@ namespace AscEmu::Realm
         std::chrono::high_resolution_clock::time_point lastPing;
     };
 
-    class ::AuthSocket;
-    class ::LogonCommServerSocket;
+    //class ::AuthSocket;
+    //class ::LogonCommServerSocket;
```

Comment:
idk why this here, for fbsd and linux clang build, it makes error, so i comment this

Reply:
![image](https://user-images.githubusercontent.com/1216225/120359875-d677ca80-c310-11eb-9e1f-e30f5ded8bf6.png)